### PR TITLE
austin.intro.1.chelsey.betenbough

### PR DESCRIPTION
**Pull request by cb10bo**

### CODE REVIEW

File: 02Lesson/01Project/index.html [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+      <h3>Lunch Specials</h3>
+        <ul>
+          <li>Grilled Cheese with Pesto</li>
+          <li>Turkey Club</li>
+          <li>Chicken Salad Sandwich</li>
+        </ul>
+        <br>
+        <br>
+      <h3>CONTACT US</h3>
+      <footer>J&B Coffee Co. (806) 796-1114 | Lubbock, TX | © 2014 J&B Coffee Co.</footer>
```
Comment (zacacollier) [inline 02Lesson/01Project/index.html:33]: - Always enclose text content in a text element (`<p>`, `<h1>`, `<a>` etc)
- For symbols like `&` you need to "escape" them with special formatting, such as `&amp;` for an ampersand. Google "& symbol html" for more info

File: 02Lesson/01Homework/index.html [hunk 2]
```diff
@@ -1,8 +1,106 @@
...
+      header {
+        display: block;
+        align-items: stretch;
+      }
+
+      table, thead, tr, td {
+        border: 4px solid black;
+        font-family: Arial;
+        font-size: 20px;
+        padding: 2px, 2px, 2px, 2px;
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:33]: No commas needed here between values.

File: 02Lesson/01Homework/index.html [hunk 3]
```diff
@@ -1,8 +1,106 @@
...
+    Chelsey Renee
+  </h1>
+
+  <p style="font-size:24px; font-family: Arial">
+    <em>"And your very flesh shall be a great poem."</em>
+  </p>
+  <br>
+  <summary>
+    <header>
+      <img src="https://static1.squarespace.com/static/52d5a9abe4b09ed2cf1f4935/t/5588a3e4e4b042906e16fdc4/1435018227847/?format=1500w"/>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:48]: Missing alt attribute

File: 02Lesson/01Homework/index.html [hunk 4]
```diff
@@ -1,8 +1,106 @@
...
+        <li>Strange Case of Dr Jekyll and Mr Hyde</li>
+      </ul>
+    </div>
+    <div>
+      <h2 style="font-family: Arial">
+        Favorite TV Shows
+      </h2>
+      <p>
+        <a href="http://www.fxnetworks.com/shows/its-always-sunny-in-philadelphia/about">
+          <img src="https://67.media.tumblr.com/d861e0433c5ba636c0597875aee63a03/tumblr_n35drbonk41toump9o1_500.gif"/>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:66]: Missing alt attribute

File: 02Lesson/01Project/index.html [hunk 5]
```diff
@@ -1,9 +1,37 @@
...
+  <body>
+    <img src="http://restaurantguidelubbock.com/wp-content/uploads/2012/07/01-J-b-coffee-co.jpg"/>
+    <h1>J&B Coffee</h1>
+    <br>
+    <h2>ABOUT US</h2>
+    <p>We love coffee and we love the Lubbock community.
+      Whether you need a study or meeting location,
+      or even a place to throw your next shower -
+      we have the space you need.</p>
+      <br>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:15]: Indentation: I would always put block elements in their own line:

``` html
<p>
  We love ....
</p>
```

Then you see immediately that the `br` tag is indented too much.

File: 02Lesson/01Project/index.html [hunk 6]
```diff
@@ -1,9 +1,37 @@
...
+      <h2>MENU</h2>
+      <h3>Signature Drinks</h3>
+        <ul>
+          <li>Lavendar Latte</li>
+          <li>Greek Treat</li>
+          <li>Americano</li>
+        </ul>
+      <h3>Lunch Specials</h3>
+        <ul>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:25]: Indentation: `ul` and `h3` should be on the same level. Put block element tags on their own line.

File: 02Lesson/01Project/index.html (hunk 1 above)
Comment (ivome) [reply 02Lesson/01Project/index.html:33]: Most of the times it makes sense to put text in a its own HTML element to be able to style it with padding, margin etc. But if you don't need it like in this case, I wouldn't add unnecessary HTML elements.

File: 02Lesson/02Project/index.html [hunk 7]
```diff
@@ -1,10 +1,45 @@
...
+      </a>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <article>
+        <h2>
+          Betenbough Homes Advanced Framing System
+        </h2>
+        <img src="http://lubbockonline.com/sites/default/files/imagecache/superphoto/13430057.jpg"/>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:28]: Missing alt attribute

File: 02Lesson/02Project/index.html [hunk 8]
```diff
@@ -1,10 +1,45 @@
...
+    <aside>
+      <img src="https://pbs.twimg.com/profile_images/469138213009776641/7wDWfX7X_400x400.jpeg" alt="New Homes"/>
+      <a href="http://www.betenbough.com/">
+        New Homes from the 120's
+      </a>
+    </aside>
+  </main>
+  <footer>
+    Copyright © 2013 Daily Planet. All Rights Reserved.
+  </footer>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:43]: Nice HTML code and formatting!

File: 03Lesson/01Project/css/style.css [hunk 9]
```diff
@@ -1 +1,27 @@
+body {
+  font-family: Arial;
+}
 
+header {
+  background-color:	#1E90FF;
+  }
```
Comment (ivome) [inline 03Lesson/01Project/css/style.css:8]: Don't indent the closing bracket.

File: 03Lesson/01Project/css/style.css [hunk 10]
```diff
@@ -1 +1,27 @@
+body {
+  font-family: Arial;
+}
 
+header {
+  background-color:	#1E90FF;
```
Comment (ivome) [inline 03Lesson/01Project/css/style.css]: Use consistent spacing. I would always use one space after the `:`

File: 03Lesson/01Project/css/style.css [hunk 11]
```diff
@@ -1 +1,27 @@
...
+
+h1 {
+  color: white;
+}
+
+a {
+  color: white;
+}
+
+.float{
```
Comment (ivome) [inline 03Lesson/01Project/css/style.css]: Add a space after the selector

**Review by ivome — COMMENTED** [review]
Good job! Nice HTML code and formatting, especially in the later projects! I only found a few very minor formatting issues. Keep it up!

**Pull request closed without merging**